projects/storefrontlib/src/cms-components/product/product-tabs/product-attributes/product-attributes.component.spec.ts

```diff
@@ -37,4 +58,11 @@ describe('ProductAttributesComponent in product', () => {
   it('should create', () => {
     expect(productAttributesComponent).toBeTruthy();
   });
+
+  it('should have rendered attribute spec using mock value', async(() => {
+    fixture.detectChanges();
+    const componentHTML = fixture.debugElement.nativeElement.outerHTML;
+    const expectedTag = '<li> mock value <span> MK </span>';
```

Comment:
I'm not a fan of the way you are testing the value, I would do something like `By.css('td:last-of-type li)` to get the element. Then you can assert that the test it "mock value MK", you can also check the first child element is the span.

Reply:
Fixed